apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -286,6 +286,16 @@ export default {
       this.rowErrors = getRowErrors(this.tableMetaData, this.rowData);
       this.saveDisabledMessage = getSaveDisabledMessage(this.rowErrors);
     },
+    getHeadingLabel(headingName: string) {
+      const column = this.tableMetaData.columns.find(
+        (column) => column.name === headingName
```

Comment:
I know it is existing code, but can you think if we indeed want column.name here or column.identifier?

Reply:
I did, in #2715 it is changed to Id because the logic behind it gets a bit simplified.